src/main/java/com/kodilla/ecommercee/domain/User.java

```diff
@@ -22,6 +22,12 @@
     @Column(name = "ID", unique = true)
     private Long id;
 
+    public User(String username, int status, int userKey) {
```

Comment:
Proszę o uzasadnienie?

Reply:
Nie zmieniałem nic w klasie User. Wygląda na to, że może pisałem na starszej wersji a potem po rebase do main się to zmieniło. W commicie poniżej usunąłem tą klasę. Proszę o akceptację.

Reply:
ja już zaakceptowałam.